addons/subscription/__openerp__.py

```diff
@@ -18,12 +18,13 @@
     * Define a subscription whose source document is the document defined as
       above. Specify the interval information and partner to be invoice.
     """,
+    'author': 'Odoo S.A.',
```

Comment:
You can get rid of default manifest fields like author, version, and installable.